reports/AC187/report_for_AC187_cleanup.rb

```diff
@@ -0,0 +1,60 @@
+require 'archivesspace/client'
+require 'json'
+require 'csv'
+require_relative '../../helper_methods.rb'
+
+aspace_staging_login()
+
+start_time = "Process started: #{Time.now}"
+puts start_time
+
+eadid = "AC187"
+resource_id = "4185"
+output_file = "#{eadid}.csv"
+ao_tree = @client.get("/repositories/4/resources/#{resource_id}/ordered_records").parsed
+#/repositories/:repo_id/resources/:id/ordered_records
+#get_all_archival_objects_for_resource(4, 4185, ['top_containers'])
+
+  CSV.open(output_file, "a",
+    :write_headers=> true,
+    :headers => ["uri", "ref_id", "title", "first_date", "container"]) do |row|
+  ao_tree['uris'].each do |ao_ref|
+    ao_uri = []
+    ao_uri << ao_ref['ref'] unless ao_ref.dig('level') == 'collection'
+    #puts ao_uri
+
+    ao_uri.each do |uri|
+      #puts uri.class
+      get_ao = @client.get(uri).parsed
+      top_containers =
+        unless get_ao['instances'].nil?
+          get_ao['instances'].each do |instance|
+            top_container =
+              if instance.dig('sub_container').nil? == false
+                then @client.get(instance['sub_container']['top_container']['ref']).parsed
```

Comment:
FYI then is not a required word in ruby.  You can drop all of these...

Reply:
Thanks!